Makefile

```
@@ -92,29 +95,33 @@ build-updater: deps
 ## Style ##
 ###########
 .PHONY: style
-style: blanks golangci-lint staticcheck no-large-files
+style: golangci-lint blanks newlines staticcheck check-service-protos no-large-files
```

```suggestion
style: golangci-lint blanks newlines check-service-protos no-large-files
```
`staticcheck` is included in golangci https://github.com/stackrox/scanner/pull/902/files#diff-6179837f7df53a6f05c522b6b7bb566d484d5465d9894fb04910dd08bb40dcc9R109

---

good point